import_craft.py

```diff
@@ -114,13 +114,14 @@ def smooth_object_meshes(self, ob):
     def read_parts_models(self, prefabs_dict, colliders, allow_no_material_mesh):
         for part in prefabs_dict.values():
             unselect_all_objects()
+
             result = import_mu.import_mu(self, bpy.context, part['mu'], False)
             if not result == {'FINISHED'}:
-                print('Warning : Error while importing file {}'.format(part['mu']))
-                return
-            part['object'] = bpy.context.scene.objects.active
-            self.smooth_object_meshes(part['object'])
-            self.rename_data_elements(part['object'])
+                print('Warning: Error while importing file {}'.format(part['mu']))
```

Comment:
Returning breaks the for loop so we skip all the following parts. 
To make the subsequent method work in this case, I added a test [here](https://github.com/sketchfab/io_object_mu/pull/7/files#diff-e93a4e5dd950597e0e3bfff88209c879R142) to ensure that we have an object associated with the part we deal with.